.travis.yml

```diff
@@ -0,0 +1,21 @@
+dist: bionic
+language: python
+python: 3.8
+
+install:
+  - pip install pyyaml anytree deprecation stringcase
```

Comment:
I’d propose to use the requirements.txt file for the installation of the dependencies.

Reply:
Yes, of course. Thanks for that feedback.  I will update it.